misago/users/signals.py

```diff
@@ -1,11 +1,22 @@
 from django.dispatch import Signal, receiver
-
+from django.contrib.auth import get_user_model
+from misago.core.utils import ANONYMOUS_IP
 
 anonymize_user_content = Signal()
 delete_user_content = Signal()
 username_changed = Signal()
-
+anonymize_old_ips = Signal()
 
```

Comment:
There should be two blank lines to separate list of defined signals from anything else.

Also please keep list of signals ordered alphabetically.